multicluster/test/e2e/fixtures.go

```diff
@@ -68,7 +69,7 @@ func teardownTest(tb testing.TB, data *MCTestData) {
 func createPodWrapper(tb testing.TB, data *MCTestData, cluster string, namespace string, name string, nodeName string, image string, ctr string, command []string,
 	args []string, env []corev1.EnvVar, ports []corev1.ContainerPort, hostNetwork bool, mutateFunc func(pod *corev1.Pod)) error {
 	tb.Logf("Creating Pod '%s' in Namespace %s of cluster %s", name, namespace, cluster)
-	if err := data.createPod(cluster, name, nodeName, namespace, ctr, image, command, args, env, ports, hostNetwork, mutateFunc); err != nil {
+	if err := data.createPod(cluster, name, nodeName, namespace, ctr, image, command, args, env, ports, hostNetwork, mutateFunc); err != nil && !apierrors.IsAlreadyExists(err) {
```

Comment:
Is this trying to fix ScaleDownMCServiceEndpoints? I investigated that failure and had PR https://github.com/antrea-io/antrea/pull/4987 to fix it. I think ignoring already exists error is not helpful as the Pod is actually being terminated, so the Pod can't be used for testing anyway even it doesn't return here. And if we are trying to create a Pod that already exists, it may mean the test logic is wrong.

Reply:
Yes. When I was testing, I found that some pods would not be deleted, so I think maybe we could reuse them. Since the bug has been fixed, I rebased the code and removed this part.